.gitignore

```diff
@@ -34,3 +34,5 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+
+.idea/
```

Comment:
pycharm ma tuhle slozku na svoje interni veci a tu nechci nikdy commitovat, tak davam do .gitignore

Reply:
Pycharm je code editor pro Python? Nebo k čemu tohle slouží?

Reply:
jo intellij maj editory pro ruzny jazyky. my mame v praci python tak delam v pycharm, ale umi to i js a ruzny dalsi plugins to ma jak vs code